### PR TITLE
Admin categories(Multi-level menu)

### PR DESCRIPTION
**Pull request by phamminhphuong**
redmine: https://edu-redmine.framgia.vn/issues/3388

### CODE REVIEW

**Comment by phamminhphuong** [comment]
ready

File: app/controllers/products_controller.rb [hunk 1]
```diff
@@ -9,6 +9,7 @@ def show
     @details = Product.show_detail(params[:id])
       .page(params[:page])
       .per Settings.size.size_page
+    @images =Image.where(product_id: @product).limit Settings.size.list_image
```
Comment (huuphuc93) [inline app/controllers/products_controller.rb]: Chuyển where(product_id: @product) thành scope đi em, sau  dấu = cách 1 space ra

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,5 @@
 module ApplicationHelper
+  def select_category_parent(id)
```
Comment (huuphuc93) [inline app/helpers/application_helper.rb]: Xóa ()

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -1,2 +1,5 @@
...
+    Category.where(parent_id: id)
```
Comment (huuphuc93) [inline app/helpers/application_helper.rb]: Xóa ()

**Comment by huuphuc93** [comment]
commented

**Pull request merged**